### PR TITLE
Rework on the dashboard

### LINKED ISSUE #455: Resolve Metric issues, : poller don't get metrics correctly
### Describe the problem you discovered

- Metric should include all txns
- when timeout, metric dashboard should have metrics have of node itself.
image

Also on the dashboard it's seems that the datas are additionned instead of being averaged

### Describe the solution you'd like

Metric poller should retrieve datas without error.
Dashboard view should have accurate values

### PR DESCRIPTION
**Pull request by samuelmanzanera**
# Description

Fix the dashboard metrics aggregation and plotting.

Fixes #455

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

By running transactions the graphs should be updated with up/down lines.

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/archethic/metrics/poller.ex [hunk 1]
```diff
@@ -9,31 +9,24 @@ defmodule Archethic.Metrics.Poller do
...
-  end
-
   def init([interval]) do
     timer = schedule_polling(interval)
 
-    state =
-      default_state()
-      |> Map.put(:interval, interval)
-      |> Map.put(:timer, timer)
+    state = %{
```
Comment (apoorv-2204) [inline lib/archethic/metrics/poller.ex:25]:
```
%{}
|>Map.put(..,..);
|>Map.put(...,...);

```

??

**Comment by apoorv-2204** [comment]
Screenshot_20220927_144258
Wow!

File: lib/archethic/metrics/poller.ex (hunk 1 above)
Comment (samuelmanzanera) [reply lib/archethic/metrics/poller.ex:25]: Explicitness is better in my opinion. 
Map.put is nice when you want to update a map but to create it, this form seems better.

**Comment by apoorv-2204** [comment]
Can we have complete granfana telemetry ?

**Comment by samuelmanzanera** [comment]
This should be managed separately.

**Review by Neylix — APPROVED** [review]

**Pull request merged**